src/NuGet.Clients/NuGet.CommandLine/Commands/InstallCommand.cs

```diff
@@ -371,9 +371,9 @@ private CommandLineSourceRepositoryProvider GetSourceRepositoryProvider()
                     resolutionContext.SourceCacheContext.NoCache = NoCache;
                     resolutionContext.SourceCacheContext.DirectDownload = DirectDownload;
 
-                    PackageNamespacesConfiguration packageNamespacesConfiguration = PackageNamespacesConfiguration.GetPackageNamespacesConfiguration(Settings);
+                    var packageSourceMappingConfiguration = PackageSourceMapping.GetPackageSourceMapping(Settings);
```

Comment:
I didn't change the variable names but focused on using `var` keyword to avoid breaking references to it. I will create a follow-up PR and fix the variable names, if needed.

Reply:
My comment is not about the `var` keyword, it's about the naming. 

It's `var packageSourceMappingConfiguration`, but should be `var packageSourceMapping`

Reply:
sure. I will fix the variable naming in the follow-up issue. https://github.com/NuGet/Client.Engineering/issues/1219